loadtest/control/utils.go

```diff
@@ -265,7 +271,7 @@ func ParseServerVersion(versionString string) (semver.Version, error) {
 // AttachFilesToPost uploads at least one file on behalf of the user, attaching
 // all uploaded files to the post.
 func AttachFilesToPost(u user.User, post *model.Post) error {
-	fileIDs, err := _attachFilesToObj(u, post.ChannelId)
+	fileIDs, err := _attachFilesToObj(u, post.ChannelId, rand.Intn(4))
```

Comment:
This should be just 4 now
```suggestion
	fileIDs, err := _attachFilesToObj(u, post.ChannelId, 4)
```